config/tasks/task_steps_config-recipe_coffee.yaml

```diff
@@ -22,182 +22,189 @@ steps:
   # - std_conf: standard deviation of confidence for both true and false
   #             examples.
   - id: 0
+    activity_id: 0      # The first step must always be background.
+    description: >-
+      Background        # The first step must always be background.
```

Comment:
I'm not sure if the trailing comment on this line will behave the way you think it will. The `description` value that is loaded from this is `"Background        # The first step must always be background."`.